### ISSUE TITLE
OpenVZ support

### ISSUE THREAD

**Issue by mrqwer88** [yadutaf/ctop#8]
Hello!

Are you planned to add OpenVZ support to ctop?

Now with openvz not work memory usage output - 
![ctop_openvz](https://cloud.githubusercontent.com/assets/7100008/7654337/fb29297e-fb24-11e4-8595-9c6003aac708.png)

Unfortunatly memory limited without cgroup mountpoints memory on OpenVZ.
Info about containers memory usage can be parse from /proc/user_beancounters 

```
     1210:  kmemsize                 15746882             18927616  9223372036854775807  9223372036854775807                    0
            lockedpages                  3964                 4008               614400               614400                    0
            privvmpages                 64129               106034              2457600              2457600                    0
            shmpages                     2897                 6435  9223372036854775807  9223372036854775807                    0
            dummy                           0                    0  9223372036854775807  9223372036854775807                    0
            numproc                        38                   81                  512                  512                    0
            physpages                   93052               107404                    0               614400                    0
            vmguarpages                     0                    0               614400  9223372036854775807                    0
            oomguarpages                 7500                 8646               614400  9223372036854775807                    0
            numtcpsock                     14                   35                64000                64000                    0
            numflock                        4                    8                 2048                 2048                    0
            numpty                          0                   10  9223372036854775807  9223372036854775807                    0
            numsiginfo                      0                   36  9223372036854775807  9223372036854775807                    0
            tcpsndbuf                  244160              3242288  9223372036854775807  9223372036854775807                    0
            tcprcvbuf                  229376              2468192  9223372036854775807  9223372036854775807                    0
            othersockbuf                90168               234176  9223372036854775807  9223372036854775807                    0
            dgramrcvbuf                     0                 6936  9223372036854775807  9223372036854775807                    0
            numothersock                   74                   87  9223372036854775807  9223372036854775807                    0
            dcachesize                6946034              7279958  9223372036854775807  9223372036854775807                    0
            numfile                       719                 1055                64000                64000                    0
            dummy                           0                    0  9223372036854775807  9223372036854775807                    0
            dummy                           0                    0  9223372036854775807  9223372036854775807                    0
            dummy                           0                    0  9223372036854775807  9223372036854775807                    0
            numiptent                      27                   27                  256                  256                    0

```

or from vzlist output

```
vzlist -H -o physpages 1212 
    122044
vzlist -o physpages 1212 -j
[
  {
    "physpages": {
      "held": 122043,
      "maxheld": 999576,
      "barrier": 0,
      "limit": 614400,
      "failcnt": 30
    }
  }
]
```

as exapmle.

**Comment by yadutaf** [comment]
Sadly, I do not have access to an OpenVZ system. If you are willing to contribute OpenVZ support I'd gladly review + merge.

**Comment by mrqwer88** [comment]
I add pull request with openvz support https://github.com/yadutaf/ctop/pull/10
How I see it work =)

**Comment by yadutaf** [comment]
Implemented by #11. Thanks.
